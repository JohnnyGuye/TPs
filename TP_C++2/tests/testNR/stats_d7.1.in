ADD 0 2015 5 8 10 5 5 N
ADD 499 2015 5 8 10 5 5 J
ADD 998 2015 5 8 10 5 5 N
ADD 1497 2015 5 8 10 5 5 N
ADD 1996 2015 5 8 10 5 5 N
ADD 2495 2015 5 8 10 5 5 V
ADD 2994 2015 5 8 10 5 5 V
ADD 3493 2015 5 8 10 5 5 R
ADD 3992 2015 5 8 10 5 5 J
ADD 4491 2015 5 8 10 5 5 V
ADD 4990 2015 5 8 10 5 5 V
ADD 5489 2015 5 8 10 5 5 V
ADD 5988 2015 5 8 10 5 5 R
ADD 6487 2015 5 8 10 5 5 R
ADD 6986 2015 5 8 10 5 5 N
ADD 7485 2015 5 8 10 5 5 N
ADD 7984 2015 5 8 10 5 5 J
ADD 8483 2015 5 8 10 5 5 R
ADD 8982 2015 5 8 10 5 5 N
ADD 9481 2015 5 8 10 5 5 N
ADD 0 2015 5 8 10 18 5 V
ADD 499 2015 5 8 10 18 5 N
ADD 998 2015 5 8 10 18 5 V
ADD 1497 2015 5 8 10 18 5 R
ADD 1996 2015 5 8 10 18 5 V
ADD 2495 2015 5 8 10 18 5 J
ADD 2994 2015 5 8 10 18 5 N
ADD 3493 2015 5 8 10 18 5 J
ADD 3992 2015 5 8 10 18 5 V
ADD 4491 2015 5 8 10 18 5 R
ADD 4990 2015 5 8 10 18 5 R
ADD 5489 2015 5 8 10 18 5 R
ADD 5988 2015 5 8 10 18 5 R
ADD 6487 2015 5 8 10 18 5 R
ADD 6986 2015 5 8 10 18 5 J
ADD 7485 2015 5 8 10 18 5 V
ADD 7984 2015 5 8 10 18 5 J
ADD 8483 2015 5 8 10 18 5 R
ADD 8982 2015 5 8 10 18 5 N
ADD 9481 2015 5 8 10 18 5 J
ADD 0 2015 5 8 10 31 5 V
ADD 499 2015 5 8 10 31 5 N
ADD 998 2015 5 8 10 31 5 V
ADD 1497 2015 5 8 10 31 5 J
ADD 1996 2015 5 8 10 31 5 R
ADD 2495 2015 5 8 10 31 5 R
ADD 2994 2015 5 8 10 31 5 N
ADD 3493 2015 5 8 10 31 5 R
ADD 3992 2015 5 8 10 31 5 V
ADD 4491 2015 5 8 10 31 5 J
ADD 4990 2015 5 8 10 31 5 J
ADD 5489 2015 5 8 10 31 5 N
ADD 5988 2015 5 8 10 31 5 N
ADD 6487 2015 5 8 10 31 5 V
ADD 6986 2015 5 8 10 31 5 J
ADD 7485 2015 5 8 10 31 5 V
ADD 7984 2015 5 8 10 31 5 N
ADD 8483 2015 5 8 10 31 5 V
ADD 8982 2015 5 8 10 31 5 N
ADD 9481 2015 5 8 10 31 5 V
ADD 0 2015 5 8 10 44 5 V
ADD 499 2015 5 8 10 44 5 J
ADD 998 2015 5 8 10 44 5 J
ADD 1497 2015 5 8 10 44 5 J
ADD 1996 2015 5 8 10 44 5 J
ADD 2495 2015 5 8 10 44 5 J
ADD 2994 2015 5 8 10 44 5 V
ADD 3493 2015 5 8 10 44 5 V
ADD 3992 2015 5 8 10 44 5 R
ADD 4491 2015 5 8 10 44 5 J
ADD 4990 2015 5 8 10 44 5 R
ADD 5489 2015 5 8 10 44 5 J
ADD 5988 2015 5 8 10 44 5 N
ADD 6487 2015 5 8 10 44 5 V
ADD 6986 2015 5 8 10 44 5 J
ADD 7485 2015 5 8 10 44 5 N
ADD 7984 2015 5 8 10 44 5 N
ADD 8483 2015 5 8 10 44 5 R
ADD 8982 2015 5 8 10 44 5 R
ADD 9481 2015 5 8 10 44 5 J
ADD 0 2015 5 8 10 57 5 N
ADD 499 2015 5 8 10 57 5 J
ADD 998 2015 5 8 10 57 5 J
ADD 1497 2015 5 8 10 57 5 R
ADD 1996 2015 5 8 10 57 5 J
ADD 2495 2015 5 8 10 57 5 J
ADD 2994 2015 5 8 10 57 5 R
ADD 3493 2015 5 8 10 57 5 R
ADD 3992 2015 5 8 10 57 5 N
ADD 4491 2015 5 8 10 57 5 J
ADD 4990 2015 5 8 10 57 5 N
ADD 5489 2015 5 8 10 57 5 R
ADD 5988 2015 5 8 10 57 5 J
ADD 6487 2015 5 8 10 57 5 V
ADD 6986 2015 5 8 10 57 5 J
ADD 7485 2015 5 8 10 57 5 J
ADD 7984 2015 5 8 10 57 5 V
ADD 8483 2015 5 8 10 57 5 J
ADD 8982 2015 5 8 10 57 5 J
ADD 9481 2015 5 8 10 57 5 J
ADD 0 2015 5 8 10 5 5 V
ADD 999 2015 5 8 10 5 5 V
ADD 1998 2015 5 8 10 5 5 R
ADD 2997 2015 5 8 10 5 5 V
ADD 3996 2015 5 8 10 5 5 V
ADD 4995 2015 5 8 10 5 5 V
ADD 5994 2015 5 8 10 5 5 N
ADD 6993 2015 5 8 10 5 5 N
ADD 7992 2015 5 8 10 5 5 R
ADD 8991 2015 5 8 10 5 5 J
ADD 0 2015 5 8 10 18 5 J
ADD 999 2015 5 8 10 18 5 N
ADD 1998 2015 5 8 10 18 5 V
ADD 2997 2015 5 8 10 18 5 N
ADD 3996 2015 5 8 10 18 5 V
ADD 4995 2015 5 8 10 18 5 V
ADD 5994 2015 5 8 10 18 5 V
ADD 6993 2015 5 8 10 18 5 R
ADD 7992 2015 5 8 10 18 5 N
ADD 8991 2015 5 8 10 18 5 V
ADD 0 2015 5 8 10 31 5 V
ADD 999 2015 5 8 10 31 5 N
ADD 1998 2015 5 8 10 31 5 R
ADD 2997 2015 5 8 10 31 5 N
ADD 3996 2015 5 8 10 31 5 N
ADD 4995 2015 5 8 10 31 5 R
ADD 5994 2015 5 8 10 31 5 V
ADD 6993 2015 5 8 10 31 5 R
ADD 7992 2015 5 8 10 31 5 N
ADD 8991 2015 5 8 10 31 5 R
ADD 0 2015 5 8 10 44 5 R
ADD 999 2015 5 8 10 44 5 R
ADD 1998 2015 5 8 10 44 5 R
ADD 2997 2015 5 8 10 44 5 V
ADD 3996 2015 5 8 10 44 5 J
ADD 4995 2015 5 8 10 44 5 V
ADD 5994 2015 5 8 10 44 5 R
ADD 6993 2015 5 8 10 44 5 V
ADD 7992 2015 5 8 10 44 5 V
ADD 8991 2015 5 8 10 44 5 N
ADD 0 2015 5 8 10 57 5 V
ADD 999 2015 5 8 10 57 5 R
ADD 1998 2015 5 8 10 57 5 R
ADD 2997 2015 5 8 10 57 5 R
ADD 3996 2015 5 8 10 57 5 R
ADD 4995 2015 5 8 10 57 5 N
ADD 5994 2015 5 8 10 57 5 V
ADD 6993 2015 5 8 10 57 5 J
ADD 7992 2015 5 8 10 57 5 R
ADD 8991 2015 5 8 10 57 5 J
ADD 0 2015 5 8 11 10 5 N
ADD 999 2015 5 8 11 10 5 N
ADD 1998 2015 5 8 11 10 5 N
ADD 2997 2015 5 8 11 10 5 V
ADD 3996 2015 5 8 11 10 5 V
ADD 4995 2015 5 8 11 10 5 V
ADD 5994 2015 5 8 11 10 5 V
ADD 6993 2015 5 8 11 10 5 N
ADD 7992 2015 5 8 11 10 5 R
ADD 8991 2015 5 8 11 10 5 V
ADD 0 2015 5 8 11 23 5 N
ADD 999 2015 5 8 11 23 5 V
ADD 1998 2015 5 8 11 23 5 V
ADD 2997 2015 5 8 11 23 5 R
ADD 3996 2015 5 8 11 23 5 V
ADD 4995 2015 5 8 11 23 5 R
ADD 5994 2015 5 8 11 23 5 V
ADD 6993 2015 5 8 11 23 5 J
ADD 7992 2015 5 8 11 23 5 J
ADD 8991 2015 5 8 11 23 5 V
ADD 0 2015 5 8 11 36 5 R
ADD 999 2015 5 8 11 36 5 J
ADD 1998 2015 5 8 11 36 5 J
ADD 2997 2015 5 8 11 36 5 R
ADD 3996 2015 5 8 11 36 5 R
ADD 4995 2015 5 8 11 36 5 N
ADD 5994 2015 5 8 11 36 5 J
ADD 6993 2015 5 8 11 36 5 R
ADD 7992 2015 5 8 11 36 5 V
ADD 8991 2015 5 8 11 36 5 R
ADD 0 2015 5 8 11 49 5 R
ADD 999 2015 5 8 11 49 5 V
ADD 1998 2015 5 8 11 49 5 V
ADD 2997 2015 5 8 11 49 5 V
ADD 3996 2015 5 8 11 49 5 J
ADD 4995 2015 5 8 11 49 5 J
ADD 5994 2015 5 8 11 49 5 N
ADD 6993 2015 5 8 11 49 5 R
ADD 7992 2015 5 8 11 49 5 J
ADD 8991 2015 5 8 11 49 5 R
ADD 0 2015 5 8 12 2 5 R
ADD 999 2015 5 8 12 2 5 R
ADD 1998 2015 5 8 12 2 5 V
ADD 2997 2015 5 8 12 2 5 N
ADD 3996 2015 5 8 12 2 5 N
ADD 4995 2015 5 8 12 2 5 N
ADD 5994 2015 5 8 12 2 5 V
ADD 6993 2015 5 8 12 2 5 N
ADD 7992 2015 5 8 12 2 5 V
ADD 8991 2015 5 8 12 2 5 R
STATS_D7 5
EXIT
