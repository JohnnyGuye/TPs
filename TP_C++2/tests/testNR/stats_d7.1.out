V 26%
J 26%
R 25%
N 23%
